packages/language-server/src/prisma-fmt/nativeTypes.ts

```diff
@@ -8,23 +8,17 @@ export interface NativeTypeConstructors {
 }
 
 export default function nativeTypeConstructors(
-  execPath: string,
   text: string,
   onError?: (errorMessage: string) => void,
 ): NativeTypeConstructors[] {
-  const result = execa.sync(execPath, ['native-types'], { input: text })
-
-  if (result.exitCode !== 0) {
-    const errorMessage =
-      "prisma-fmt error'd during getting available native types.\n"
```

Comment:
This error message might have gotten lost in the conversion. @tomhoule

Reply:
In `previewFeatures.ts` it was kept - and should probably here as well.

Reply:
#965

applied labels so it doesn't get lost but I intend to do this today.